magpy/absolutes.py

```diff
@@ -1652,24 +1658,26 @@ def absoluteAnalysis(absdata, variodata, scalardata, **kwargs):
 
         for elem in filelist:
             head, tail = os.path.split(elem)
-            try:
-                date = dparser.parse(tail,fuzzy=True)
-                datelist.append(datetime.strftime(date,"%Y-%m-%d"))
-            except:
-                print("absoluteAnalysis: Found date problem in file: %s" % tail)
-                failinglist.append(elem)
-
+            if "starttime=" in elem:
+                pos = elem.find("starttime=")+10
+                epos = pos + 10
```

Comment:
Not sure if dates will always be 10 characters?  It may be better to look for an ampersand following `starttime=`